Add Nerd font Icons & mph

dianaw353 opened the pull request:
I have rewritten the code based on you fork this time around alongside of translating the sunrise and sunset variables with the help of google and if its wrong someone else can fix them

bjesus commented:
Thanks @dianaw353 ! it's really much easier to review PRs when they do one thing and one thing only. seems like this adds nerd fonts, adds support for mph instead of kmh, and does something with sunrise and sunset. do you think there's a chance to split these changes? i can review this like it is but it'll just take me longer.

bjesus commented on src/main.rs:
```diff
@@ -152,33 +175,49 @@ fn main() {
-            tooltip += &format!("{}, ", lang.tomorrow());
-        }
+        tooltip += &format!("{}, ", lang.today());
+        tooltip += &format!("{}, ", lang.tomorrow());
```
this looks wrong, now we're printing things like "Today, Tomorrow, 2024-04-21" for every day.

dianaw353 commented:
The clock style I just added a space in between the number and am/pm to make it look better. The additional change with sunrise and sunset is just to make it easier for people in other languages to read without the need to translate them.

dianaw353 commented:
Alright, I made a cleaner version of the get_weather_icon so it should be easier to add more in the future when needed(still could improve a bit more) and removed the translations for sunrise/sunset as it was not needed and implemented the use_nerd_font for sunrise and sunset to go along with the rest of the patch.

dianaw353 commented:
closing this as ive exasperated them into their own pull requests
